cpp/models/abm/parameters.h

```diff
@@ -512,10 +514,13 @@ struct AgeGroupGotoSchool {
  * @brief The set of AgeGroups that can go to work.
  */
 struct AgeGroupGotoWork {
-    using Type = std::set<AgeGroup>;
-    static Type get_default(AgeGroup /*size*/)
+    using Type = CustomIndexArray<bool, AgeGroup>;
+    static Type get_default(AgeGroup num_agegroups)
     {
-        return std::set<AgeGroup>{AgeGroup(2), AgeGroup(3)};
+        auto a = Type(num_agegroups, false);
+        a[AgeGroup(2)] = true;
+        a[AgeGroup(3)] = true;
```

Comment:
This could be a bit problematic if the number of age groups is lower than 3. This was already a problem earlier I guess, but could be worse now due to the indexing. Perhaps we could return a more standard default case and rather set this parameter in each simulation?

Reply:
I am worried about this as well. The usage is a bit awkward as well, first setting the whole array to false and some of them to true (see the abm unit tests). what would the default be? all false?

Reply:
I guess all false makes sense, and then set up an initialisation script that sets parameters (also others potentially) to values that we commonly use in tests/simulations. This is beyond this PR though, so I am inclined to leave it as it is and open a new issue for setting up parameters separately. This is a step that we wanted to to anyway at some point.

Reply:
created issue #869